scripts/zip.sh

```diff
@@ -1,16 +1,19 @@
 #!/bin/bash
 
+TARGET=$1
+FILENAME=WikipediaKaiOS$TARGET
+
 rm -rf Wikipedia
 rm -rf dist
-rm app.zip
+rm $FILENAME.zip
 
 mkdir Wikipedia
 
 node scripts/manifest.js > Wikipedia/manifest.webapp
-cross-env INSTRUMENTATION=1 npm run build
+LANG_LIST=$TARGET INSTRUMENTATION=1 npm run build
```

Comment:
When `cross-env` was added here it was also added for the `dev` and `deploy` commands in package.json.

https://github.com/wikimedia/wikipedia-kaios/pull/230/files

Should we also update (remove/replace/add) package.json with `LANG_LIST`?

Reply:
I've removed `cross-env` from here because it fails for me locally and the primary task of this script is to generate the bundle in CI on linux. No need to be cross env.

I haven't updated the scripts in package.json because I don't this the `jio` lang list should be the default in dev.

Reply:
ya, that's fine. I wasn't able to produce the manifest file correctly even using the cross-env on Windows.